unit_tests/RenameEnumMembersUnitTests.d

```diff
@@ -0,0 +1,272 @@
+/**
+ * Copyright: Copyright (c) 2017 Wojciech Szęszoł. All rights reserved.
+ * Authors: Wojciech Szęszoł
+ * Version: Initial created: Aug 08, 2017
+ * License: $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0)
+ */
+import std.stdio;
+import Common;
+import dstep.translator.Translator;
+import dstep.translator.Enum;
+
+void assertRenameEnumMembers(
+    string[] input,
+    string[] expected,
+    string file = __FILE__,
+    size_t line = __LINE__)
+{
+    import core.exception;
+    import std.algorithm.comparison;
+    import std.format;
+    import std.range;
+
+    auto actual = renameEnumMembers(input[0], input[1..$]);
+
+    if (!equal(expected[1..$], actual))
+    {
+        auto formatString = "\nExpected:\n%s\nActual:\n%s\n";
+        auto message = format(formatString, expected, chain([input.front], actual));
+        throw new AssertError(message, file, line);
+    }
+}
+
+unittest
+{
+    assertRenameEnumMembers(
+        ["PrefixFoo", "PrefixFOO", "PrefixBAR", "PrefixBAZ"],
+        ["PrefixFoo", "FOO", "BAR", "BAZ"]);
```

Comment:
Wouldn't it be more clear if the enum names (not the members) were passed as separate arguments from the members?

Reply:
Oh, whatever.